browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1308,6 +1308,99 @@ until they verify, or until 90 days have passed.
       <message name="IDS_NIGHT_MODE_ENABLE" desc="Menu item for enabling dark mode for webcontent.">
         Enable "Night Mode" (Experimental)
       </message>
+      <message name="IDS_MENU_EDIT" desc="NTP widget menu item">
+        Edit Stack
+      </message>
+      <message name="IDS_MENU_REMOVE" desc="NTP widget menu item">
+        Remove Widget
+      </message>
+      <message name="IDS_MENU_LEARN_MORE" desc="NTP widget menu item">
+        Learn More
+      </message>
+      <message name="IDS_MENU_REFRESH_DATA" desc="NTP widget menu item">
+        Refresh Data
+      </message>
+      <message name="IDS_MENU_DISCONNECT" desc="NTP widget menu item">
+        Disconnect
+      </message>
+      <message name="IDS_BTC_BALANCE" desc="Binance widget BTC balance text">
+        <ph name="BTC_BALANCE">%1$s<ex>0.0</ex></ph>BTC
+      </message>
+      <message name="IDS_USD_BALANCE" desc="Binance widget USD balance text">
+        ≈ $<ph name="USD_BALANCE">%1$s<ex>0.0</ex></ph>
+      </message>
+      <message name="IDS_CURRENCY_ADDRESS_TEXT" desc="Binance deposit coin address text">
+        <ph name="COIN_ADDRESS">%1$s</ph> Deposit Address
+      </message>
+      <message name="IDS_CURRENCY_MEMO_TEXT" desc="Binance deposit coin memo text">
+        <ph name="COIN_MEMO">%1$s</ph> Deposit Memo
+      </message>
+      <message name="IDS_BUY_CRYPTO" desc="Binance buy section text.">
+        Buy Crypto
+      </message>
+      <message name="IDS_COM_TEXT" desc="Binance buy section .com text.">
+        .com
+      </message>
+      <message name="IDS_US_TEXT" desc="Binance buy section .us text.">
+        .us
```

Comment:
Maybe we can hardcode that one and .com as we don't need them to translate ever?